src/imitation/util/networks.py

```diff
@@ -117,6 +117,57 @@ def forward(self, x: th.Tensor) -> th.Tensor:
         return (x - self.running_mean) / th.sqrt(self.running_var + self.eps)
 
 
+class EMARunningNorm(RunningNorm):
+    """Similar to RunningNorm but uses an exponentially weighting."""
+
+    running_mean: th.Tensor
+    running_var: th.Tensor
+
+    def __init__(
+        self,
+        num_features: int,
+        lamb: float = 0.05,
+        r: float = 0.05,
+        eps: float = 1e-5,
+    ):
+        """Builds EMARunningNorm.
+
+        Args:
+            num_features: Number of features; the length of the non-batch dim.
+            lamb: value between (0, 1) update rate for the mean
+            r: value between (0, 1) update rate for the variance
+            eps: small constant for for numerical stability.
+        """
+        super().__init__(num_features, eps=eps)
+        self.lamb = lamb
+        self.r = r
+
+    def update_stats(self, batch: th.Tensor) -> None:
+        """Update `self.running_mean` and `self.running_var`.
```

Comment:
you update count too right? But it's perhaps not actually used for anything beyond an auxiliary statistic.

Reply:
I wanted to keep the interface the same for everything.